### PR TITLE
Added a vote to change the stability fee structure

### PR DESCRIPTION
**Pull request by hexonaut**
From here: https://forum.makerdao.com/t/signal-request-change-the-stability-fee-structure/2380

**Note**: This is a ranked choice poll. There are no previous examples of this yet, so I made up the format. I can adjust based on feedback.

### CODE REVIEW

File: governance/polls/Proposal - Change the Stability Fee Structure - May 18 2020.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Change the Stability Fee Structure - May 18 2020
+summary: Signal your support to change the structure of the weekly Stability Fee votes.
+discussion_link: https://forum.makerdao.com/t/2380
+poll_rules: The voter may select to vote for one or more of the poll options or they may elect to abstain from the poll entirely
```
Comment (jparklev) [inline governance/polls/Proposal - Change the Stability Fee Structure - May 18 2020.md:5]:
```suggestion
vote_type: Ranked Choice IRV
```
this is the only change you need to make to mark this poll as ranked choice :) (see [this PR](https://github.com/makerdao/community/pull/540/files) for another example of poll designed to be RC)

**Comment by jpritikin** [comment]
My comments were ignored?

**Comment by hexonaut** [comment]
Which comments?

**Comment by jpritikin** [comment]
I opened a review in this pull request, 
Screenshot from 2020-05-18 11-23-02

**Comment by hexonaut** [comment]
Very odd... I still don't see your comments.

**Comment by jpritikin** [comment]
Oh, maybe my fault. I didn't `submit` the review until now?

**Comment by hexonaut** [comment]
Yeah I see it now. Too bad, I like your suggestions.

**Pull request merged**